main.py

```diff
@@ -4,9 +4,6 @@
 from discord.ext import commands
 from dotenv import load_dotenv
 
-# Creates the db if it doesn't exist
-import util.db_builder
```

Comment:
Why remove this?
This is added so that you no longer need to run db_builder.py when starting up a new instance of the bot. This line does that for you. (https://github.com/Arborym/Pax-Academia/blob/63f9ed8c56f708bc5c520814937b7ce18de5abe9/util/db_builder.py#LL1C1-L5C1)

Reply:
When python imports something, it automatically runs the entire file. (That's why you see `if __name__ == __main__` so often). I use this feature to run the script db_builder.py in the util folder automatically every time the bot starts. If someone creates a new instance of the bot, all tables of the db are created if they do not exist yet.

So although it is never referenced in main.py, it is actually the equivalent of running db_builder.py every time the bot starts.